packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -728,6 +734,55 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		return { age: revertAge };
 	}
 
+	/**
+	 * Recursively creates a revertible for the given commit and its ancestors.
+	 */
+	private forkRevertible(
+		view: RervertibleTarget,
+		commit: GraphCommit<SharedTreeChange>,
+		data: CommitMetadata,
+	): Revertible {
+		const forkRevertibleFunction = this.forkRevertible.bind(this);
+		const revertibleCommit = this.revertibleCommitBranches.get(commit.revision);
+
+		const forked: Revertible = {
+			get status(): RevertibleStatus {
+				return revertibleCommit === undefined
+					? RevertibleStatus.Disposed
+					: RevertibleStatus.Valid;
+			},
+			revert: (release: boolean = true) => {
```

Comment:
To stick with the terminology in the interface. Or was there a reason to call it `release`?
```suggestion
			revert: (dispose: boolean = true) => {
```